tools/benchmark_tool/openvino/tools/benchmark/main.py

```diff
@@ -299,12 +315,12 @@ def set_throughput_streams():
                 keys = benchmark.ie.get_metric(device, 'SUPPORTED_CONFIG_KEYS')
                 logger.info(f'DEVICE: {device}')
                 for k in keys:
-                    logger.info(f'  {k}  , {exe_network.get_config(k)}')
+                    logger.info(f'  {k}  , {benchmark.ie.get_config(device, k)}')
```

Comment:
The same question @myshevts

Reply:
same as above this should be the network, not the plugin